src/components/ProductImage/ProductImage.js

```diff
@@ -10,13 +9,13 @@ class ProductImage extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      selectedImage: '',
+      selectedImage: ''
     };
   }
 
   clickHandler(selectedImage) {
     this.setState({
-      selectedImage,
+      selectedImage
     });
   }
 
```

Comment:
`renderBigImage ` is not a render function at all. It needs to be renamed. And the ternary operator can be used inside for returning image src. And I will attach 2 interesting links for an explanation of line 43 and line 57 - why can't you do that